katello/katello-restore

```diff
@@ -36,9 +36,9 @@ def restore
   puts "Done.\n"
 
   puts "Restoring backend data"
-  `tar --selinux --overwrite -xf pulp_data.tar -C /`
-  `tar --selinux --overwrite -xzf mongo_data.tar.gz -C /`
-  `tar --selinux --overwrite -xzf pgsql_data.tar.gz -C /`
+  `tar --selinux --overwrite --listed-incremental=pulp.snar -xf pulp_data.tar -C /`
```

Comment:
Won't this create a `No such file or directory` error if `--skip-pulp-content` is used during the backup? It shouldn't be an issue but the user may not know that.